nix/all-engines.nix

```diff
@@ -17,13 +17,13 @@ in
     name = "prisma-engines";
     inherit src;
 
-    buildInputs = [ pkgs.openssl ];
+    buildInputs = [ pkgs.openssl.out ];
     nativeBuildInputs = with pkgs; [
+      cargo
       git # for our build scripts that bake in the git hash
-      perl # for openssl-sys
-      pkg-config
       protobuf # for tonic
-      cargo
+      openssl.dev
+      pkg-config
```

Comment:
These changes make it so we don't need perl to discover openssl anymore. pkg-config on its own works great.